### ISSUE TITLE
Please Rewrite the Docs and include more examples.

### ISSUE THREAD

**Issue by alixsep** [neutralinojs/neutralinojs#535]
Neutralino.js is great! but unfortunately the docs are extremely poor written. I would like to see:

1. A much better written docs
2. A part of the doc that explains source code and how it works
As result, 1 will only lead to more people getting into using neutralino and 2 will make contributing much easier!

**Comment by jaswafford1211** [comment]
Yes, I agree. I think a page should also be focused on the modes `"browser": {}` and `"cloud": {}`.

I think Neutralino.js could be a big break in writing desktop apps. As a whole, I am very pleased with Neutralino.js over Electron.

**Comment by lkn849** [comment]
I agree too! I think the "cloud" option looks very promising
